feelings/groups/forms.py

```diff
@@ -24,10 +23,10 @@ def clean_email_or_username(self):
         data = self.cleaned_data['email_or_username']
         try:
             self.invitee = models.User.objects.get(
-                Q(email=data)|Q(username=data)
+                Q(email=data) | Q(username=data)
```

Comment:
No it's not Kenneth the first violates PEP8 is E227 according to Anaconda in sublimetext.